mapiproxy/servers/default/emsmdb/dcesrv_exchange_emsmdb.c

```diff
@@ -1542,12 +1542,14 @@ static enum MAPISTATUS dcesrv_EcDoAsyncConnectEx(struct dcesrv_call_state *dce_c
 	struct dcesrv_handle		*handle;
 	struct dcesrv_endpoint		*item;
 	bool				found;
+	const char			*email;
 
 	OC_DEBUG(3, "exchange_emsmdb: EcDoAsyncConnectEx (0xe)\n");
 
 	/* Step 0. Ensure incoming user is authenticated */
 	if (!dcesrv_call_authenticated(dce_call)) {
 		OC_DEBUG(1, "No challenge requested by client, cannot authenticate");
+	failure:
```

Comment:
I would prefer you to revert the `failure: and goto failure` change because I don't like it and it is detrimental to readability imho